cpp/benchmarks/pairwise_linestring_distance.cu

```diff
@@ -78,7 +78,7 @@ std::tuple<rmm::device_vector<cartesian_2d<T>>, rmm::device_vector<int32_t>> gen
   std::vector<cartesian_2d<T>> points(num_points);
 
   auto random_walk_func = [segment_length](auto const& prev, auto const& rad) {
-    return cartesian_2d<T>{prev.x + segment_length * rad.x, prev.y + segment_length * rad.y};
+    return cartesian_2d<T>({prev.x + segment_length * rad.x, prev.y + segment_length * rad.y});
```

Comment:
Please don't include this change in this PR. I don't agree that this is the right way to fix this issue, and we haven't even root-caused the issue yet. I have been discussing it with @trxcllnt

Reply:
Understood!